shared/utils/__tests__/logic.spec.ts

```diff
@@ -78,7 +72,7 @@ describe('Logic validation', () => {
         ],
         _id: MOCK_LOGIC_ID,
         logicType: LogicType.ShowFields,
-      } as IShowFieldsLogicSchema
+      } as unknown as FormLogic
```

Comment:
might want to update these mocks too instead of casting as unknown. All the types are now using primitives and should be easily created